Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -9128,6 +9153,7 @@
 				SDKROOT = iphoneos;
 				SUPPORTS_MACCATALYST = NO;
 				SWIFT_VERSION = 5.0;
+				TEST_HOST = "$(BUILT_PRODUCTS_DIR)/KickDebug.app/$(BUNDLE_EXECUTABLE_FOLDER_PATH)/KickDebug";
```

Comment:
This is the change that was required to get the unit tests to work. It's fairly innocuous looking in Xcode. Annoyingly, I can't find any good documentation around this switch, and why we didn't have it switched on before - my guess is that it might increase the runtime for `Library-Tests`, which I'll keep an eye on.
<img width="1308" alt="Screenshot 2024-02-14 at 11 05 18 AM" src="https://github.com/kickstarter/ios-oss/assets/146007185/14aca856-7f94-43e5-a1c2-576bf87c6281">